include/xtensor/xmime.hpp

```diff
@@ -132,9 +139,16 @@ namespace xt
                 compute_2d_row(out, printer, expr, edgeitems, idx_str, row_idx);
             }
             out << "<tr>";
-            for (std::size_t column_idx = 0; column_idx < 2 * edgeitems + 1; ++column_idx)
+            for (std::size_t column_idx = 0; column_idx < nb_ellipsis; ++column_idx)
             {
-                out << "<td><center>...</center></td>";
+                if (column_idx == edgeitems && nb_ellipsis != last_dim)
+                {
+                    out << "<td><center>⋱</center></td>";
+                }
+                else
+                {
+                    out << "<td><center>⋮</center></td>";
```

Comment:
Can you try  `\u22EE` or `\u22ee` instead of putting the unicode character in the source?

This could have a comment saying that it is the unicode character for a vertical ellipsis.